dockstore-webservice/src/main/java/io/dockstore/webservice/resources/EntryResource.java

```diff
@@ -248,6 +249,9 @@ public void exportToORCID(@Parameter(hidden = true, name = "user") @Auth User us
             LOG.error("ORCID auth URL is likely incorrect");
             throw new CustomWebApplicationException("Could not export to ORCID: Dockstore ORCID integration is not set up correctly.", HttpStatus.SC_INTERNAL_SERVER_ERROR);
         }
+        if (orcidByUserId.isEmpty()) {
+            throw new CustomWebApplicationException("ORCID account is not linked to user account", HttpStatus.SC_INTERNAL_SERVER_ERROR);
```

Comment:
I think this should be a user, 4xx, error... it's not Dockstore's fault somebody invoked this API without having linked their ORCID account.